app/src/actions/search.js

```diff
@@ -0,0 +1,32 @@
+import { GET_SEARCH_RESULTS } from '../constants';
+import 'whatwg-fetch';
+
+const url = 'https://skytruth-pleuston.appspot.com/v1/tilesets/tms-format-2015-2016-v1/search/';
```

Comment:
If we reuse this URL somewhere else, we could make a global variable such as `API_URL`.